src/strategy/actions/TradeStatusAction.cpp

```diff
@@ -235,6 +237,17 @@ bool TradeStatusAction::CheckTrade()
 
     if (!botMoney && !playerMoney)
         return true;
+	
+	// Player just wants money from the bot
+    if (sPlayerbotAIConfig->allowRandomBotsAtm && !isGettingItem && !isGivingItem && botMoney > playerMoney)
```

Comment:
Isn't this the wrong way around? If the option is enabled it calls the player a thief, doesn't it?

Reply:
The bot calls the player a thief because he's forced to give the player money.

Reply:
Yes but ` (sPlayerbotAIConfig->allowRandomBotsAtm &&` <- if the option is **enabled** we call the player a thief. Shouldn't it be ` (!sPlayerbotAIConfig->allowRandomBotsAtm &&`?

Reply:
Oh dear, I am not reading it right. You are correct, that's my bad. It accepts the trade, not deny it. Sorry.